GOT-StudentVersion/GOT-StudentVersion/Controllers/GOTETableViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  GOTETableViewController.swift
+//  GOTE Project
+//
+//  Created by Mariel Hoepelman on 8/18/19.
+//  Copyright © 2019 Mariel Hoepelman. All rights reserved.
+//
+
+import UIKit
+
+class GOTETableViewController: UITableViewController {
+    var episode = GOTEpisode.allEpisodes
+    
+    @IBOutlet weak var searchBarOulet: UISearchBar!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+//        searchBarOulet.delegate = self
+    }
+
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 7
+    }
+    
+    // GOTEpisode.episodesBySeason(season: section + 1).count works by calling episodesbySeason function in model. episodesbySeason function takes a season as an argument and returns the episodes that correspont to a specific season by filtering the property by season number.
+    //In the function below, the return is the count of episodes in a season. the function called receives a season argument that is an Int, in this case the value is a section number + 1. e.g. for section 0, the value of season is 0 + 1 = 1, it returns season1.count. For season 2, returns section 1 + 1 = 2, season2.count and so on.
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return GOTEpisode.episodesBySeason(season: section + 1).count
+    }
+    
+    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 150
+    }
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch section{
```

Comment:
So here instead of hardcoding these values, you can refer to the ones you model holds
 return "Season \(section + 1)"